app/Locale/English.php

```diff
@@ -506,7 +506,7 @@ public function get_locale(): array
 			'CAMERA_DATE' => 'Camera date',
 			'NEW_PASSWORD' => 'new password',
 			'ALLOW_UPLOADS' => 'Allow uploads',
-			'RESTRICTED_ACCOUNT' => 'Restricted account',
+			'ALLOW_USER_SELF_EDIT' => 'Allow user modifications',
```

Comment:
I don't like the translation. First and for all, it does not make clear that the capability is restricted to management of the _own_ user account. The way how it is written one could assume that it allows to manage any user account. Secondly, my mental picture of a "user modification" imagines a tattoo or piercing studio :rofl:

Maybe

```suggestion
			'ALLOW_USER_SELF_EDIT' => 'Allow self-management of user account',
```

?